Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,142 @@
+[!!Monitor authorizations](../Operation/01_ManageAuthorizations.md)
+
+# LIST (Authorizations)
+
+![LIST (Authorizations)](../../Assets/Screenshots/Payments/Authorizations/LISTAuthorizations.png "[LIST (Authorizations)]")
+
+**Authorizations**
+
+- *VIEW*  
+    Click the drop-down list to select the view. All created views are displayed in the drop-down list. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *VIEW* drop-down list to display the context menu and create a view.   
+
+    - ![Points](../../Assets/Icons/Points01.png "[Points]") (Points)      
+        Click this button to the right of the *View* drop-down list to display the context menu. The following menu entries are available:
+
+        - ![Create](../../Assets/Icons/Plus06.png "[Create]") create  
+            Click this entry to create a view. The *Create view* window is displayed, see [Create view](#create-view).
+
+        - ![Rename](../../Assets/Icons/Edit02.png "[Rename]") rename  
+            Click this entry to rename the selected view. The *Rename view* window is displayed, see [Rename view](#rename-view). This menu entry is only displayed if you have selected a view.
```

Comment:
Zu Passiv habe ich das gefunden: https://actindo.atlassian.net/wiki/spaces/DOC/pages/96927782/Additional+guidelines
Use passive voice when talking about windows, tabs, views etc. For example, a window is closed, a view is displayed but never a tab closes, or a tab appears.

Ich weiß, was du denkst, aber ich finde es umständlich, jetzt Varianten hinzufügen. Wir müssen uns es gut überlegen. Das kommt Tausende Mal vor...